### DISCUSSION TITLE
Resistive Bridge Perfomance

### DISCUSSION THREAD

**Discussion by AdrielAgostini** [jankae/LibreVNA#252] [Q&A]
Hi everybody, 
I'm studying Electrical Engineering and I'm trying to make an RF coupler with the passband of this VNA, so I think that the implementation here will work for me. Are there any measurements of the performance of the coupler alone?
Thanks.

**Comment by jankae** [comment]
Sorry, I don't have any measurement of just the coupler. When I designed this VNA, I didn't have any test equipment that could take such measurement, as a result I didn't bother testing the coupler on its own.

**Reply by AdrielAgostini** [reply]
Oh, and do you design the coupler by your own or you got it from other side?

**Reply by jankae** [reply]
It is just a resistive coupler, so not much to design. The main benefit is that it is small and simple. Of course you are paying for this with increased losses. So yes, the schematic and layout are my own designs (if you can call copying a well known combination of resistors a design)
